Gems/Atom/Utils/Code/Tests/PngFileTests.cpp

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/Utils/PngFile.h>
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzCore/Math/Color.h>
+#include <AzCore/std/containers/array.h>
+#include <AzCore/IO/FileIO.h>
+#include <AzFramework/IO/LocalFileIO.h>
+
+namespace UnitTest
+{
+    using namespace AZ::Utils;
+
+    class PngFileTests
+        : public AllocatorsFixture
+    {
+    protected:
+        AZStd::string m_testImageFolder;
+        AZStd::string m_tempPngFilePath;
```

Comment:
We should be using AZ::IO::Path for path manipulation

```suggestion
        AZ::IO::Path m_testImageFolder;
        AZ::IO::Path m_tempPngFilePath;
```